server/controllers/appController.js

```diff
@@ -0,0 +1,68 @@
+const db = require('../database.js');
+const appController = {};
+
+// Create a new application in the database
+appController.createApp = (req, res, next) => {
+  const company = req.body.company;
```

Comment:
We'll also need a field to identify the user with whom this app is associated. We should be able to read it off the cookie.